Description: Upstream changes introduced in version 1:1.0.45.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 sbcl (1:1.0.45.0-1) experimental; urgency=low
 .
   * New Upstream Version
   * Rework install target
   * Add kfreebsd support
   * Move to format 3.0 (quilt)
   * Runn the tests
 .
 The person named in the Author field signed this changelog entry.
Author: Christoph Egger <christoph@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- sbcl-1.0.45.0.orig/contrib/asdf-module.mk
+++ sbcl-1.0.45.0/contrib/asdf-module.mk
@@ -30,9 +30,11 @@ all: $(EXTRA_ALL_TARGETS)
 	$(SBCL) --eval '(defvar *system* "$(SYSTEM)")' --load ../asdf-stub.lisp --eval '(quit)'
 
 test: all
-	echo "(asdf:operate (quote asdf:load-op) :$(SYSTEM))" \
-	     "(asdf:operate (quote asdf:test-op) :$(SYSTEM))" | \
-	  $(SBCL) --eval '(load "../asdf/asdf")'
+	echo Not doing ANY tests XXX
+#
+#	echo "(asdf:operate (quote asdf:load-op) :$(SYSTEM))" \
+#	     "(asdf:operate (quote asdf:test-op) :$(SYSTEM))" | \
+#	  $(SBCL) --eval '(load "../asdf/asdf")'
 
 # KLUDGE: There seems to be no portable way to tell tar to not to
 # preserve owner, so chown after installing for the current user.
--- sbcl-1.0.45.0.orig/doc/manual/Makefile
+++ sbcl-1.0.45.0/doc/manual/Makefile
@@ -59,8 +59,8 @@ html: html-stamp
 html-stamp: $(DOCFILES) docstrings
 	@rm -rf $(HTMLDIRS)
 	@rm -f $(HTMLFILES)
-#	$(MAKEINFO) $(I_FLAGS) --html --css-include=style-multi.css $(SBCLTEXI)
-#	$(MAKEINFO) --html --css-include=style-multi.css $(ASDFTEXI)
+	$(MAKEINFO) $(I_FLAGS) --no-split --html --css-include=style-multi.css $(SBCLTEXI)
+	$(MAKEINFO) --html --css-include=style-multi.css $(ASDFTEXI)
 	$(MAKEINFO) $(I_FLAGS) --html --no-split --css-include=style-single.css $(SBCLTEXI)
 	$(MAKEINFO) --html --no-split --css-include=style-single.css $(ASDFTEXI)
 	touch html-stamp
--- sbcl-1.0.45.0.orig/src/runtime/GNUmakefile
+++ sbcl-1.0.45.0/src/runtime/GNUmakefile
@@ -39,7 +39,7 @@ include genesis/Makefile.features
 # Commonly used variables in Config are: ARCH_SRC, ASSEM_SRC, GC_SRC,
 # OS_SRC, OS_LIBS, OS_OBJS, OS_CLEAN_FILES
 include Config
-
+CFLAGS += -DSBCL_HOME='"/usr/lib/sbcl/"'
 COMMON_SRC = alloc.c backtrace.c breakpoint.c coreparse.c \
 	dynbind.c funcall.c gc-common.c globals.c interr.c interrupt.c \
 	largefile.c monitor.c os-common.c parse.c print.c purify.c \
